src/lib/htmlInputPropsUtils.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash'
+
+export const htmlInputProps = [
+  // REACT
+  'selected', 'defaultValue', 'defaultChecked',
+
+  // LIMITED HTML PROPS
+  'autoCapitalize', 'autoComplete', 'autoFocus', 'checked', 'form', 'max', 'maxLength', 'min', 'multiple',
+  'name', 'pattern', 'placeholder', 'readOnly', 'required', 'step', 'type', 'value',
+
+  // Heads Up!
+  // Do not pass disabled, it duplicates the SUI CSS opacity rule.
+  // 'disabled',
+]
+
+export const htmlInputEvents = [
+  // EVENTS
+  // keyboard
+  'onKeyDown', 'onKeyPress', 'onKeyUp',
+
+  // focus
+  'onFocus', 'onBlur',
+
+  // form
+  'onChange', 'onInput',
+
+  // mouse
+  'onClick', 'onContextMenu',
+  'onDrag', 'onDragEnd', 'onDragEnter', 'onDragExit', 'onDragLeave', 'onDragOver', 'onDragStart', 'onDrop',
+  'onMouseDown', 'onMouseEnter', 'onMouseLeave', 'onMouseMove', 'onMouseOut', 'onMouseOver', 'onMouseUp',
+
+  // selection
+  'onSelect',
+
+  // touch
+  'onTouchCancel', 'onTouchEnd', 'onTouchMove', 'onTouchStart',
+]
+
+export const htmlInputPropNames = [...htmlInputProps, ...htmlInputEvents]
+
+export const omitHTMLInputProps = (props, htmlProps = htmlInputPropNames) => _.omit(props, htmlProps)
```

Comment:
Lodash has perf issues with `_.omit` and it is also planned to be removed in v5, #864 .  How about we move these two pick/omit methods into a single reduce that returns a pair of objects?  Something like:

```jsx
const [htmlInputProps, rest]  = partitionHTMLInputProps(props)
```
Given:
```jsx
const htmlInputProps = ['selected', ...]

const getHTMLInputProps = props => htmlInputProps.reduce((acc, next) => {
  acc[+!(next in props)] = props[next]
  return acc
}, [])
```